web/src/pages/desktop/menu/index.tsx

```diff
@@ -67,23 +68,31 @@ export const Menu = () => {
           <Settings />
           <Fullscreen />
 
-          <div
-            className="mr-1 flex h-[30px] w-[30px] cursor-pointer items-center justify-center rounded text-neutral-300 hover:bg-neutral-700 hover:text-white"
-            onClick={() => setIsMenuOpen((o) => !o)}
-          >
-            <XIcon size={20} />
-          </div>
+          <Tooltip title={t('menu.collapse')} placement="bottom">
+            <div
+              className="mr-1 flex h-[30px] w-[30px] cursor-pointer items-center justify-center rounded text-neutral-300 hover:bg-neutral-700 hover:text-white"
+              onClick={() => setIsMenuOpen((o) => !o)}
+            >
+              <XIcon size={20} />
+            </div>
+          </Tooltip>
         </div>
 
         {!isMenuOpen && (
-          <div
-            className="flex h-[30px] w-[50px] items-center justify-center rounded bg-neutral-800/50 text-white/50 hover:bg-neutral-800 hover:text-white"
-            onClick={() => setIsMenuOpen((o) => !o)}
-          >
-            <MenuIcon />
-          </div>
+          <Tooltip title={t('menu.expand')} placement="bottom">
+            <div
+              className="flex h-[30px] w-[50px] items-center justify-center rounded bg-neutral-800/50 text-white/50 hover:bg-neutral-800 hover:text-white"
+              onClick={() => setIsMenuOpen((o) => !o)}
+            >
+              <MenuIcon />
+            </div>
+          </Tooltip>
         )}
       </div>
     </div>
   );
 };
+function msg(arg0: string): any {
```

Comment:
Where is this used from?

In which scenarios would it be called?

Reply:
Thanks for pointing this out.
I think somehow my editor generated this!